modules/database/src/Database.ts

```diff
@@ -36,6 +36,7 @@ import { CustomEndpointController } from './controllers/customEndpoints/customEn
 import { status } from '@grpc/grpc-js';
 import path from 'path';
 import { isEmpty } from 'lodash';
+import { CallContext } from 'nice-grpc-common';
```

Comment:
nice-grpc is not a database dependecy. Remove it and use better types for the call object, probably through ts-proto